docs: typos and grammar fixes (#180)
nuxt/image: docs/content/en/api/options.md

## `sizes`

> List of predefined sizes for responsive images. These sizes will be used to generate a resized and optimized version of an image.

Reviewer: Not sure which is the case?
```suggestion
List of predefined sizes for responsive images. These sizes will be used to generate resized and optimized versions of an image.
```

## `intersectOptions`

> The module uses [IntersectionObserver](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API) to detect whether the images are in Viewport or not. Use this option to modify IntersectionObserver options.

Reviewer: ```suggestion
The module uses [IntersectionObserver](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API) to detect whether the images are in the viewport or not. Use this option to modify IntersectionObserver options.
```

## `static`

> - `dir`: The root directory of the all images. By default nuxt image looks in the `static` dir to find original images.

Reviewer: ```suggestion
- `dir`: The root directory for your images. By default `@nuxt/image` looks in the `static` dir to find original images.
```

> - `clearCache`: The ipx has a caching strategy to clear cached images to reduce massive disk usages. You can schedule the cache cleaning job using the `clearCache` option in the provide options. By default this cron job is disabled.

Reviewer: ```suggestion
- `clearCache`: If enabled, `ipx` can run a cron job to clear cached images to help prevent massive disk usage. By default this is disabled.
```

- `cacheDir`: The directory to store the cached images.

> - `sharp`: Modify default behavior of image optimizer. Note that if you change this option, you need to clear the ipx cache.

Reviewer: ```suggestion
- `sharp`: Modify the default behavior of the image optimizer. Note that if you change this option, you will need to clear the ipx cache.
```
